dependencies.yaml

```diff
@@ -0,0 +1,124 @@
+# Dependency list for https://github.com/rapidsai/dependency-file-generator
+files:
+  all:
+    output: conda
+    matrix:
+      cuda: ["11.5"]
+      arch: [x86_64]
+    includes:
+      - build
+      - cudatoolkit
+      - develop
+      - doc
+      - notebook
+      - run
+      - test_python
+channels:
+  - rapidsai
+  - rapidsai-nightly
+  - dask/label/dev
+  - conda-forge
+  - nvidia
+dependencies:
+  build:
+    common:
+      - output_types: [conda, requirements]
+        packages:
+          - cmake>=3.23.1,!=3.25.0
+          - cuda-python >=11.7.1,<12.0
+          - cython>=0.29,<0.30
+          - ninja
+          - scikit-build>=0.13.1
+      - output_types: [conda]
+        packages:
+          - c-compiler
+          - cxx-compiler
+          - rapids-build-env=23.02.*
```

Comment:
@ajschmidt8 Did you say this package was no longer needed on a different PR?

Reply:
Yes. For this PR, the intention is just to replace the manually managed `conda/environment` files with the dfg. I figured we can remove these packages in the follow-up GHA PR.

If we remove them before, it could break the environment. In the GHA PR, we can remove them and then identify which dependencies we need to re-add after they're removed.

Reply:
Okay, that sounds fine to me. We'll clean up later.